hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/dml/TestInsertTable.scala

```diff
@@ -430,53 +430,6 @@ class TestInsertTable extends HoodieSparkSqlTestBase {
    })
   }
 
-  test("Test Insert Into None Partitioned Table strict mode with no preCombineField") {
```

Comment:
test is failing because we overwrite the payload in provideshoodieconfig i think